### ISSUE TITLE
Configuration reference

### ISSUE THREAD

**Issue by sverhagen** [rhuss/docker-maven-plugin#132]
It'd be great if there was an overall configuration reference. All the configuration options are [well documented, one by one](https://github.com/rhuss/docker-maven-plugin/blob/master/doc/manual.md), but I find it sometimes hard to piece together what goes where. The `plugin.xml` (which I guess is generated off of the mojos) is a start, but (presumably due to generics) does not go down the hierarchy all the way. E.g. it stops at `images` as being a `List`, while in fact it contains specific options. I'm not asking for comprehensive documentation, just an overview of the configuration elements, such as on the top of [this](http://maven.apache.org/ref/3.2.5/maven-model/maven.html).

**Comment by rhuss** [comment]
Good idea. I'm already thinking about switching the manual to a richer format (e.g. like something [gitbook](https://www.gitbook.com/), so that I can e.g. use tables for the reference manual. The automatic generation of the documentation currently doesn't work properly and I'm not a big fan of the automatically generated documentation.
